src/dotnet/UI.Blazor/Services/ReloadUI.cs

```diff
@@ -21,7 +21,7 @@ public virtual void Reload(bool clearCaches = false)
             Log.LogWarning("Reloading...");
             try {
                 if (clearCaches)
-                    await ClearCaches();
+                    await ClearCaches().ConfigureAwait(false);
```

Comment:
`NavigationManager` is used further, so -> `.ConfigureAwait(true)`.

Reply:
done